04.ls/lib/file.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module LS
+end
+
+class LS::File
```

Comment:
```suggestion
module Ls
  class File
```

みたいに書くことが多いですね。
ただLSモジュールを使うのはここだけなので、それであれば単純に

```suggestion
class LsFile
```

みたいなクラス名にした方がシンプルな気がします